tools/link.py

```diff
@@ -2491,9 +2491,11 @@ def modularize():
       src += 'export default %s;\n' % settings.EXPORT_NAME
   elif not settings.MINIMAL_RUNTIME:
     src += '''\
-if (typeof exports === 'object' && typeof module === 'object')
+if (typeof exports === 'object' && typeof module === 'object') {
   module.exports = %(EXPORT_NAME)s;
-else if (typeof define === 'function' && define['amd'])
+  // This default export allows TS to import this commonjs style module.
+  module.exports.default = %(EXPORT_NAME)s;
```

Comment:
Do we both of these though?  It seems a little odd..

Reply:
I wonder if its worth include that link here, or elaborating on the comment.   e.g. "This might seem redundant but without this it is not possible to ...".